SmartTutor/Service/Recommenders/KnowledgeBasedRecommender.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SmartTutor.ContentModel;
+using SmartTutor.Repository;
+using System.Collections.Generic;
+
+namespace SmartTutor.Service.Recommenders
+{
+    class KnowledgeBasedRecommender : IRecommender
```

Comment:
I really like the idea you had when you suggested the IRecommender interface, and in particular how a HybridRecommender could implement it to compose multiple IRecommenders.  Well done 🙂